Makefile

```diff
@@ -4,17 +4,15 @@ install:
 	composer install --no-interaction
 
 clean:
-	rm -rf vendor/ dist/
+	rm -rf vendor/ dist/ composer.lock .php_cs.cache .phpunit.result.cache
 
-test: install
+test: install format
```

Comment:
format changes files and fixes them. However, in previous version `--dry-run` is applied to avoid file changes.

Reply:
Fixed!